packages/frontend/src/components/Header/Header.tsx

```diff
@@ -7,6 +7,7 @@ import { MenuBar } from '../MenuBar';
 import { AltinnLogo } from './AltinnLogo';
 import { SearchBar } from './SearchBar';
 import styles from './header.module.css';
+import { getSearchStringFromQueryParams } from '../../pages/Inbox/queryParams';
```

Comment:
Ville kanskje ha laget en todo på å refaktorere eller flytte disse query-funksjonene ut av Inbox. Det er ikke logisk for meg at dette bor i Inbox akkurat nå.

Reply:
https://github.com/digdir/dialogporten-frontend/issues/771